pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -254,167 +254,152 @@
+               :root {
+                 --white: rgb(51, 51, 51);
+                 --black100: rgba(255,255,255,1);
+                 --black05: rgba(255,255,255,0.05);
+                 --black20: rgba(255,255,255,0.2);
                }
              }
              '''
     ==
     ;body
-      ;div#main
-        ;div#inner
-          ;p:"Urbit ID"
-          ;input(value "{(scow %p our)}", disabled "true", class "mono");
-          ;p:"Access Key"
-          ;p.f9.gray2
-            ; Get key from Bridge, or
-            ;span.mono.pr1:"+code"
-            ; in dojo
-          ==
-          ;form(action "/~/login", method "post", enctype "application/x-www-form-urlencoded")
-            ;input
-              =type  "password"
-              =name  "password"
-              =placeholder  "sampel-ticlyt-migfun-falmel"
-              =class  "mono"
-              =autofocus  "true";
-            ;input(type "hidden", name "redirect", value redirect-str);
-            ;button(type "submit"):"Continue"
-          ==
-        ==
-        ;footer.absolute.w-100
-          ;div.relative.w-100.tr.tc-ns
-            ;p.pr2.pr0-ns.pb2
-              ;a(href "https://bridge.urbit.org", target "_blank")
-                ;span.dn.dib-ns.pr1:"Open"
-                ; Bridge ↗
-              ==
-              ;a
-                =href  "https://urbit.org/using/install/#id"
-                =style  "margin-left: 8px; color: #2aa779;"
-                =target  "_blank"
-                ; Purchase
-                ;span.dn.dib-ns.pr1:"an Urbit ID"
-                ; ↗
-              ==
+      ;p:"Urbit ID"
+      ;input(value "{(scow %p our)}", disabled "true", class "mono");
+      ;p:"Access Key"
+      ;form(action "/~/login", method "post", enctype "application/x-www-form-urlencoded")
+        ;input
+          =type  "password"
+          =name  "password"
+          =placeholder  "sampel-ticlyt-migfun-falmel"
+          =class  "mono"
+          =required  "true"
+          =minlength  "27"
+          =maxlength  "27"
+          :: =pattern  "((\w{6})-?){4}"
+          =autofocus  "true";
+        ;input(type "hidden", name "redirect", value redirect-str);
+        ;+  ?:  failed
+          ;span.failed
+            ;svg(xmlns "http://www.w3.org/2000/svg", viewBox "0 0 12 12")
+              ;circle(cx "6", cy "6", r "5.5");
+              ;line(x1 "3.27", y1 "3.27", x2 "8.73", y2 "8.73");
+              ;line(x1 "8.73", y1 "3.27", x2 "3.27", y2 "8.73");
             ==
+            Key is incorrect
           ==
-        ==
+          ;span;
+        ;button(type "submit"):"Continue"
       ==
     ==
+    ;script:'''
+            var failSpan = document.querySelector('.failed');
+            if (failSpan) {
+              document.querySelector("input[type=password]").addEventListener('keyup', function (event) {
+                failSpan.style.display = 'none';
+              });
+            }
+            '''
```

Comment:
Looks like it still needs the "Access Bridge" etc. links at bottom?

Reply:
https://github.com/urbit/landscape/issues/154#issuecomment-733989711

> Note that in the latest rev, the bottom links have been removed

Reply:
Welp you got me